core/db/20-schema.sql

```diff
@@ -122,6 +122,7 @@ ALTER TABLE ONLY messages
 
 CREATE TABLE inventory (
   call character varying(32) NOT NULL,
+  icn character varying(32),
```

Comment:
So you will want to put this into the migrations call because it will not be used on staging and production servers. It would be a call similar to [this](https://github.com/BYU-ODH/hlrdesk/blob/master/core/db/migration/migrate-db.sh#L18)